### ISSUE TITLE
[Bug]: If UDP 443 outbound is blocked on the machine running the master, peer connection establishment for the data channel is failed in v1.8.1

### ISSUE THREAD

**Issue by H-Tomoki** [awslabs/amazon-kinesis-video-streams-webrtc-sdk-c#1817]
### Please confirm you have already done the following

- [X] I have searched the repository for related/existing bug reports
- [X] I have all the details the issue requires

### Describe the bug

I couldn't reopen Issue #1594, so I am creating a new one. 
The original issue was occurred in v1.7.3, but v.1.8.1 also has the same problem. 
Specifically, in a network environment where the Master side's outbound UDP port 443 is blocked, it is failed to connect a peer connection when using only data channels. 
In the same network configuration as this issue, the connection is successfully established when either only the media channel is used or when both media and data channels are used.

This issue is related to the following issue and may be resolved by resolving the issue #1816

### Expected Behavior

The connection should establish consistently, regardless of whether data channels or media channels are used.

### Current Behavior

In certain network environments, a connection is not established when only the data channel is used, but when the data channels and media channels are used together.

### Reproduction Steps

For detailed steps to reproduce the issue, please refer to the original Issue #1594.

By modifying to include the Trickle ICE option even when only using the data channel, the connection was successfully established.  
The difference in behavior between using media channels and this case may be due to the presence or absence of the Trickle ICE option. More information is provided in the relevant issue: Issue #1816

### WebRTC C SDK version being used

v1.8.1

### Compiler and Version used

gcc 9.4.0

### Operating System and version

Ubuntu 20.04

### Platform being used

Linux

**Comment by jdelapla** [comment]
Given the suspicion that this has the same root issue as issue #1816 , can you check if this fix (https://github.com/awslabs/amazon-kinesis-video-streams-webrtc-sdk-c/pull/1813) resolved this issue as well.

**Comment by H-Tomoki** [comment]
@jdelapla 
As previously mentioned, adding the trickle ICE option allows for a successful connection even in the problematic network environment. However, if the trickle ICE setting on the Viewer is turned off, the trickle ICE option is not added, resulting in a failed connection. It seems to me that a more fundamental solution may be needed.

**Comment by H-Tomoki** [comment]
@jdelapla
Yes, turning off trickle ICE in the Viewer's config results in a failed connection.
As stated in #1816 , when only using the data channel, turning off trickle ICE in the config does not add the trickle ICE flag to the offer, resulting in the same outcome as before the fix.

**Comment by hassanctech** [comment]
@H-Tomoki Apologies on this yes you are correct, I have opened a PR to address this:  https://github.com/awslabs/amazon-kinesis-video-streams-webrtc-sdk-c/pull/1842
